roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -0,0 +1,74 @@
+---
+# defaults file for openshift_prometheus
+openshift_prometheus_state: present
+
+openshift_prometheus_namespace: prometheus
+
+openshift_prometheus_replicas: 1
+openshift_prometheus_node_selector: {"region":"infra"}
+
+# images
```

Comment:
Will the default images be changed to other values in the future?

Reply:
Initially, I will change `openshift_prometheus_image_alertbuffer` to official upstream image `openshift/prometheus-alert-buffer` that was created yesterday.
In the future, image tags/versions might change.